src/pyedb/configuration/cfg_operations.py

```diff
@@ -84,12 +84,13 @@ def apply(self):
         """Imports operation information from JSON."""
         if self.op_cutout:
             polygon_points = self._pedb.cutout(**self.op_cutout.get_attributes())
-            if not "pyedb_cutout" in self._pedb.stackup.all_layers:
+            if "pyedb_cutout" not in self._pedb.stackup.all_layers:
                 self._pedb.stackup.add_document_layer(name="pyedb_cutout")
                 self._pedb.modeler.create_polygon(polygon_points, layer_name="pyedb_cutout", net_name="pyedb_cutout")
 
             # create a polygon on pyedb layer
 
     def get_data_from_db(self):
         self.op_cutout = CfgCutout(self._pedb)
-        return {"cutout": self.op_cutout.get_data_from_db()}
+        if self.op_cutout.get_data_from_db():
+            return {"cutout": self.op_cutout.get_data_from_db()}
```

Comment:
```suggestion
        data_from_db = self.op_cutout.get_data_from_db()
        if data_from_db:
            return {"cutout": data_from_db}
```
No point in calling that twice I think.